deps/build.jl

```diff
@@ -378,39 +384,31 @@ function main()
 
         if !use_artifacts
```

Comment:
I think this is not needed. As it will never hit `use_artufacts` case below.

Reply:
Thanks for catching this!